src/Components/ExperimentOne/ExperimentOneUserForm.js

```diff
@@ -43,11 +44,9 @@ function ExperimentOneUserForm(props){
                 Lets find your unique character something fun to do.
                 How many friends do you wanna enjoy with:
                     <select onChange={handleNumberOfParticipants}>
-                    <option value="1">1</option>
-                    <option value="2">2</option>
-                    <option value="3">3</option>
-                    <option value="4">4</option>
-                    <option value="5">5</option>
+                       {listOfParticipants.map (participant => 
```

Comment:
Formatting is inconsistent, might want to try and set up pretter or eslint to keep things tidy

Reply:
I finally managed to sort this out(hopefully 👀). Despite having the softwares installed, I had select them as my default formatter. So things should be better now